src/main/kotlin/gg/flyte/christmas/minigame/games/Spleef.kt

```diff
@@ -254,6 +254,22 @@ class Spleef : EventMiniGame(GameConfig.SPLEEF) {
         snowmen.forEach { it.remove() }
         bees.forEach { it.remove() }
 
+        // reset game state
+        overviewTasks.clear()
+        floorLevelBlocks = listOf(
+            Util.fillArena(110, Material.SNOW_BLOCK),
+            Util.fillArena(98, Material.SNOW_BLOCK),
+            Util.fillArena(86, Material.SNOW_BLOCK)
+        ).flatten()
+        gameTime = 0
+        doubleJumps.clear()
+        snowmen.clear()
+        bees.clear()
+        bottomLayerMelted = false
+        unlimitedJumpTickData = 0 to 0
+        powerfulSnowballTickData = 0 to 0
+        donationEventsEnabled = false
+
```

Comment:
Wouldn't all this be handled by java's GC?

Reply:
Do you make a new instance of the class every time a game starts? If so then yeah I basically copied those from the top of the class where fields get defined, otherwise no this has to be done

Reply:
All games played are new instances of `EventMiniGame`